comments-on-conformance.md

```diff
@@ -1,16 +1,15 @@
+2. In the WCAG 2 conformance model, a [success criterion is satisfied](https://www.w3.org/TR/WCAG22/#dfn-satisfies) if “the success criterion does not evaluate to ‘false’ when applied to the page”.  If the success criterion is about something that does not exist on the page being evaluated (e.g. a success criterion requiring captioning of audio, but there is no audio) the success criterion is automatically satisfied; some consider the success criterion as “not applicable” but regardless it is satisfied.  This approach is central to the way the success criteria in WCAG are structured and worded.
+3.  WCAG 2 conformance is evaluated for full pages and complete processes.
+4.  In WCAG 2, when conformance relies upon accessibility features of the platform (i.e. browser for web content) or on assistive technologies, WCAG 2 requires that there exists browsers and assistive technologies that work with the product (i.e., web page content).  That is, conformance with WCAG 2 requires that the approaches used are [supported by assistive technologies](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported).
+5.  WCAG 2 allows part of a page to not conform if the same information and functionality is available elsewhere on the page in a conforming manner.  However, WCAG 2 identifies four success criteria that must be met on all areas of the page because they can interfere with the user’s ability to access and use other parts of the page:
     *   [1.4.2 Audio Control](http://www.w3.org/TR/WCAG22/#audio-control);
     *   [2.1.2 No Keyboard Trap](http://www.w3.org/TR/WCAG22/#no-keyboard-trap);
     *   [2.2.2 Pause, Stop, Hide](http://www.w3.org/TR/WCAG22/#pause-stop-hide).
     *   [2.3.1 Three Flashes or Below Threshold](http://www.w3.org/TR/WCAG22/#three-flashes-or-below-threshold);
 
-Also, as noted in the Introduction, it wasn't possible to unambiguously carve up software into discrete pieces, and so the unit of evaluation for non-web software is the whole software program. As with any software testing this can be a very large unit of evaluation, and methods similar to standard software testing might be used.
+As noted in the [Introduction](#introduction), it isn’t possible to unambiguously divide software into discrete pieces.  The unit of evaluation for non-web software is the whole software program.  As with any software testing this can be a very large unit of evaluation, and methods similar to standard software testing may be used.
```

Comment:
```suggestion
As noted in the [Introduction](#introduction), it isn’t possible to unambiguously divide software into discrete pieces.  Therefore, the unit of evaluation for non-web software is the whole software program.  As with any software testing this can be a very large unit of evaluation, and methods similar to standard software testing may be used.
```